supervisor/supervisor.go

```diff
@@ -331,11 +331,15 @@ func (s *Supervisor) restore() error {
 		id := d.Name()
 		container, err := runtime.Load(s.stateDir, id, s.shim, s.timeout)
 		if err != nil {
-			return err
+			logrus.WithFields(logrus.Fields{"error": err, "id": id}).
+				Warnf("containerd: failed to load container")
+			continue
 		}
 		processes, err := container.Processes()
 		if err != nil {
-			return err
+			logrus.WithFields(logrus.Fields{"error": err, "id": id}).
```

Comment:
Actually, this can't fail (https://github.com/docker/containerd/blob/v0.2.x/runtime/container.go#L268), the routine always succeed.

Should I just change its definition?